src/movies.js

```diff
@@ -1,15 +1,65 @@
 // Iteration 1: All directors? - Get the array of all directors.
+function getAllDirectors(movies) {
+    let directors = movies.map(function(i) {
+        return i.director;
+    }) 
+    return directors;
+}
+
```

Comment:
Use clear variable names (eg. movie) instead of i, it will make you code more readable for you and other programmers.